api/src/main/java/org/jmisb/api/klv/KlvParser.java

```diff
@@ -13,6 +17,73 @@ public class KlvParser {
 
     private KlvParser() {}
 
+    /**
```

Comment:
Ideally the javadoc would describe the difference in error handling between this method and `parseBytes`. I believe your intention is for this method to swallow and aggregate parsing exceptions coming from inside the individual messages, but still throw in the case of malformed top-level messages. Also please add a `@throws KlvParseException` to the javadoc re-iterating when you'd expect to see one of these.